packages/sdk/src/client.ts

```diff
@@ -506,6 +631,44 @@ export class AcrossClient {
     }
   }
 
+  /**
+   * Get a swap quote for a given set of parameters. See {@link getSwapQuote}.
+   * @param params - See {@link GetSwapQuoteParams}.
+   * @returns See {@link SwapApprovalApiResponse}.
+   */
+  async getSwapQuote(
+    params: MakeOptional<GetSwapQuoteParams, "logger" | "apiUrl">,
+  ): Promise<SwapApprovalApiResponse> {
+    try {
+      const quote = await getSwapQuote({
+        ...params,
+        logger: params?.logger ?? this.logger,
+        apiUrl: params?.apiUrl ?? this.apiUrl,
+      });
+      return quote;
+    } catch (e) {
+      if (
+        !this.tenderly?.simOnError ||
+        !(e instanceof AcrossApiSimulationError)
+      ) {
+        throw e;
+      }
+
+      const { simulationId, simulationUrl } = await this.simulateTxOnTenderly({
+        networkId: params.originChainId.toString(),
```

Comment:
As mentioned in one of the above comments, this can either be the origin or the destination chain ID. But if we force `skipOriginTxEstimation: true`, we might assume that this is always the destination chain ID